vars/RedHat.yml

```diff
@@ -2,3 +2,5 @@
 java: "{{ es_java | default('java-1.8.0-openjdk.x86_64') }}"
 default_file: "/etc/sysconfig/elasticsearch"
 es_home: "/usr/share/elasticsearch"
+package_manager: "{% if ansible_os_family == 'RedHat' and ansible_distribution_major_version >= '8' %}dnf{% else %}yum{% endif %}"
```

Comment:
`ansible_os_family` should not be required here because this file is only included in `tasks/main.yml` using `ansible_os_family` to derive the filename.

Reply:
Agreed. I primarily didn't want to diverge too far from existing statements in e.g. Debian.yml which has the same unnecessary check. But I'll remove it here as it's been discussed (I'm leaving it in Debian.yml for now though as I haven't tested there)

Reply:
Actually I've just removed all that and gone with `ansible_facts.pkg_mgr` to simplify things.